.github/CODEOWNERS

```diff
@@ -250,6 +248,8 @@
 /comp/forwarder/orchestrator @DataDog/agent-metrics-logs
 /comp/forwarder/orchestrator/orchestratorinterface @DataDog/agent-metrics-logs
 /comp/metadata/packagesigning @DataDog/agent-platform
+/comp/trace/etwtracer @DataDog/windows-agent
```

Comment:
Any thoughts on isolating this in a dedicated package `traceetwtracer` for example?

The reason I'm asking is because with this change, the agent-apm team would co-own this package which wasn't the case prior to this PR. This happens bc agent-apm already owns `/comp/trace`.

Unless there is value in having the agent-apm team co-own it, I'd prefer to keep the ownership 100% for the windows-agent team as it's always been for this package.

Reply:
As discussed offline, this is likely not an issue since - the agent-apm team got pinged bc /comp/trace was changed and the ownership rules were added in the same PR.